stubs/tensorflow/tensorflow/autodiff.pyi

```diff
@@ -0,0 +1,61 @@
+from _typeshed import Incomplete
+from builtins import bool as _bool
+from collections.abc import Generator, Mapping, Sequence
+from contextlib import contextmanager
+from types import TracebackType
+from typing import overload
+from typing_extensions import Self
+
+import tensorflow as tf
+from tensorflow import Tensor, UnconnectedGradients, Variable
+from tensorflow._aliases import ContainerGradients, ContainerTensors, ContainerTensorsLike, Gradients, TensorLike
+
+class ForwardAccumulator:
+    def __init__(self, primals: Tensor, tangents: Tensor) -> None: ...
+    def jvp(
```

Comment:
This also has `__enter__` and `__exit__` methods, are those worth adding?

Reply:
Since the class is meant to be used as a context manager, and the methods are present for `GradientTape`, I've added them in f3f3cc9b0.